### PR TITLE
fix(graph) fix export with empty metric

### PR DESCRIPTION
**Pull request by loiclau**
#5756

### CODE REVIEW

File: www/include/views/graphs/exportData/ExportCSVServiceData.php [hunk 1]
```diff
@@ -87,6 +90,11 @@ function get_error($str){
 
 # Order by timestamp
 ksort($datas);
+foreach ($datas as $key => $data){
```
Comment (kduret) [inline www/include/views/graphs/exportData/ExportCSVServiceData.php]: need space before "{"

File: www/include/views/graphs/exportData/ExportCSVServiceData.php [hunk 2]
```diff
@@ -97,7 +105,11 @@ function get_error($str){
 foreach ($datas as $ctime => $tab) {
     print $ctime . ";" . date("Y-m-d H:i:s", $ctime);
     foreach($tab as $metric_value) {
-        printf(";%f", $metric_value);
+        if($metric_value !== ''){
```
Comment (kduret) [inline www/include/views/graphs/exportData/ExportCSVServiceData.php]: need space before "(" and "{"

File: www/include/views/graphs/exportData/ExportCSVServiceData.php [hunk 3]
```diff
@@ -97,7 +105,11 @@ function get_error($str){
...
+            printf(";%f", $metric_value);
+        } else {
+            printf(";");
```
Comment (kduret) [inline www/include/views/graphs/exportData/ExportCSVServiceData.php]: use print instead of printf

**Comment by ganoze** [comment]
Does it need to be merged on 2.8.x and master too ?

**Comment by loiclau** [comment]
yes

**Pull request merged**